a10_octavia/controller/worker/tasks/a10_database_tasks.py

```diff
@@ -933,6 +933,21 @@ def execute(self, vthunder):
                           'due to: {}'.format(str(e)))
 
 
+class GetLoadBalancerListForDeletion(BaseDatabaseTask):
+
+    def execute(self, vthunder, loadbalancer):
+        try:
+            if vthunder:
+                loadbalancers_list = self.loadbalancer_repo.get_all_ohter_lbs_in_project(
```

Comment:
Typo. Should be `get_all_other_lbs_in_project` instead of `get_all_ohter_lbs_in_project`

Reply:
thanks
Will fix in another PR